source/MRMesh/MRObjectLoad.cpp

```diff
@@ -189,16 +189,28 @@ tl::expected<std::vector<std::shared_ptr<MR::Object>>, std::string> loadObjectFr
         else
             result = tl::make_unexpected( res.error() );
     }
-    else if ( !SceneFileFilters.empty() && filename.extension() == SceneFileFilters.front().extension.substr( 1 ) )
+    else if ( !SceneFileFilters.empty() && ( filename.extension() == SceneFileFilters.front().extension.substr( 1 ) ||
+                                             filename.extension() == SceneFileFilters.back().extension.substr( 1 ) ) )
```

Comment:
mb add separate block for each extension? (or function like `deserializeObjectTreeFromAnySupportedFormat`)